src/donor_dashboard/templates/donor_dashboard/list.html

```diff
@@ -51,49 +51,137 @@ <h2 class="subtitle mt-4">
+                                        <div class="media-right">
+                                            <span class="tag is-primary is-light is-medium mt-1">In Active</span>
+                                        </div>
+                                        <div class="media-right">
+                                            <span class="tag is-primary is-light is-medium mt-1">{{ org.org_type }}</span>
+                                        </div>
+                                    </div>
+                                    <div class="content">
+                                        <div class="columns">
+                                            <div class="column is-full-mobile is-half-tablet">
+                                                <!-- djlint:off -->
+                                                <div class="button is-link is-light has-text-centered"
+                                                    style="white-space: normal;
+                                                            word-wrap: break-word;
+                                                            overflow-wrap: break-word">{{ org.org_address }}</div>
+                                            </div>
+                                            <div class="column is-full-mobile is-half-tablet has-text-centered">
+                                                <p style="word-wrap: break-word; overflow-wrap: break-word;">{{ org.org_email }}</p>
+                                                <p>{{ org.org_contact_number }}</p>
+                                                <a href="{{ org.org_website }}"
+                                                style="word-wrap: break-word;
+                                                        overflow-wrap: break-word">{{ org.org_website }}</a>
+                                                <!-- djlint:on -->
+                                            </div>
                                         </div>
                                     </div>
                                 </div>
+                                <footer class="card-footer mt-auto">
+                                    <a href="{% url 'donor_dashboard:manage_organization' org.org_id %}" class="card-footer-item">Manage Organization</a>
+                                </footer>
                             </div>
-                            <footer class="card-footer mt-auto">
-                                <a href="{% url 'donor_dashboard:manage_organization' org.org_id %}" class="card-footer-item">Manage Organization</a>
-                            </footer>
                         </div>
-                    </div>
                 {% endfor %}
             {% else %}
-                <p class="box">You are not a part of any organizations yet!!</p>
+                <p class="box">No In Active Organizations!!</p>
             {% endif %}
         </div>
+        </div>
     </div>
+    <script>
```

Comment:
Can you please shift this to a new javascript file under static?

Reply:
Thanks @arsalananwar11!!
Updated accordingly